src/components/AssetSearch/AssetSearch.tsx

```diff
@@ -51,6 +51,7 @@ export const AssetSearch = ({ onClick, filterBy }: AssetSearchProps) => {
             {...register('search')}
             type='text'
             placeholder='Search'
+            autoFocus // eslint-disable-line jsx-a11y/no-autofocus
```

Comment:
If we use autofocus, why do we have this rule activated?

Reply:
good question !